python/paddle/fluid/contrib/mixed_precision/fp16_utils.py

```diff
@@ -116,7 +116,7 @@ def _insert_cast_op(block, op, idx, src_dtype, dest_dtype):
                         persistable=False,
                         stop_gradient=in_var.stop_gradient)
 
-                    block._insert_op(
+                    block._insert_op_without_sync(
```

Comment:
Better add a cast before` for loop`?

Reply:
why add a cast ?
Or you mean add a sync before loop?